src/components/ApplicationList.tsx

```
@@ -109,7 +109,19 @@ const ApplicationList: React.FC<ApplicationListProps> = ({
 			a.localeCompare(b, undefined, { sensitivity: 'base' })
 		);
 	}, [groupedApplications]);
-
+	const getActionLabel = (
+		status: string | null,
+		t: (key: string) => string
+	): string => {
+		if (
+			status === 'application initiated'
+		) {
```

ESLint

[error] Replace `⏎↹↹↹status·===·'application·initiated'⏎↹↹` with `status·===·'application·initiated'`

(prettier/prettier)